Prox/Prox/Utilities/AnalyticsUtilities.swift

```diff
@@ -61,11 +61,16 @@ public struct AnalyticsEvent {
     static let PARAM_ACTION       = "action"
     static let NUM_CARDS          = "num_cards"
     static let CARD_INDEX         = "card_index"
+    static let FOREGROUND         = "foreground"
```

Comment:
Yes! I updated the new strings to match, so this should be okay. We never got any of this data anyway, which is why I made a bunch of changes.